codec/builtin/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description 'Codec Builtin: Deephaven builtin codec implementations'
+
+dependencies {
+    implementation project(":codec-base"),
+                   project(":DataStructures"),
+                   project(":engine-query-constants"),
+                   project(":Util")
```

Comment:
I'm sad that we need dependency on Util, it looks like it's only so `io.deephaven.util.EncodingUtil` can be used. Also, it looks like `EncodingUtil` is _only_ used by these codecs... it probably makes sense to move `EncodingUtil` into here? (We could leave in same namespace if we think it's valuable externally; if not, we could make it an implementation detail)